locales/ja-JP.yml

```diff
@@ -139,6 +139,8 @@ flagAsBot: "Botとして設定"
 flagAsBotDescription: "このアカウントがプログラムによって運用される場合は、このフラグをオンにします。オンにすると、反応の連鎖を防ぐためのフラグとして他の開発者に役立ったり、Misskeyのシステム上での扱いがBotに合ったものになります。"
 flagAsCat: "Catとして設定"
 flagAsCatDescription: "このアカウントが猫であることを示す場合は、このフラグをオンにします。"
+flagShowTimelineReplies: "Show replies in timeline"
+flagShowTimelineRepliesDescription: "Show users replying to other users in the timeline"
```

Comment:
```suggestion
flagShowTimelineRepliesDescription: "オンにすると、タイムラインにユーザーのノート以外にもそのユーザーの他のノートへの返信を表示する。"
```

Reply:
In english: "When turned on, will not only show notes of a user, but also replies of that user to other notes".